ui/components/app/connected-sites-list/connected-sites-list.component.js

```diff
@@ -39,11 +40,20 @@ export default class ConnectedSitesList extends Component {
                 {this.getSubjectDisplayName(subject)}
               </span>
             </div>
-            <i
-              className="fas fa-trash-alt connected-sites-list__trash"
-              title={t('disconnect')}
-              onClick={() => onDisconnect(subject.origin)}
-            />
+            {isSnap ? (
```

Comment:
Rather than make the "Disconnect" button look different for snaps v.s. the connected sites page, can we instead change this to display the full word "Disconnect" in both places? Or use the trash can for now and worry about this later? It seems silly for this component to be passed an `isSnap` prop for this sole purpose. Nothing about snaps makes "Disconnect" preferable to a trash can icon.

Reply:
Also, this is a pre-existing mistake that we can fix in a separate PR, but we should use clickable elements for clickable things. Anchors and buttons are built-in elements that the browser already understands are meant to be interacted with, so they automatically get focused by keyboards and interpreted as interactive by screen readers and other assistive technologies. Patching `onClick` handlers onto divs makes accessibility unnecessarily complicated.